lib/WebService/MinFraud/Validator.pm

```diff
@@ -78,6 +78,11 @@ sub _build_request_schema_definition {
                 optional => {
                     user_agent      => '//str',
                     accept_language => '//str',
+                    session_age     => '//num',
+                    session_id      => {
+                        type   => '//str',
+                        length => { min => 1, max => 255, },
+                    },
```

Comment:
I don't see any tests for the validation and we only seem to use the `full-request.json` in an author test.